File: WOT-iOS/WOTData/WOTData/model/subclasses/VehicleprofileArmor+CoreDataProperties.m
//
//  VehicleprofileArmor+CoreDataProperties.m
//  WOTData
//
//  Created on 8/28/18.
//  Copyright © 2018. All rights reserved.
//
//

#import "VehicleprofileArmor+CoreDataProperties.h"

@implementation VehicleprofileArmor (CoreDataProperties)

+ (NSFetchRequest<VehicleprofileArmor *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"VehicleprofileArmor"];
}

@dynamic front;
@dynamic rear;
@dynamic sides;
@dynamic vehicleprofileArmorListHull;
@dynamic vehicleprofileArmorListTurret;

@end

File: WOT-iOS/WOTApi/WOTApi/response/managedObjects/autogenerated/VehicleprofileArmor+CoreDataProperties.swift
//
//  VehicleprofileArmor+CoreDataProperties.swift
//  WOTData
//
//  Created by Pavel Yeshchyk on 5/1/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

import CoreData

public extension VehicleprofileArmor {
    @nonobjc class func fetchRequest() -> NSFetchRequest<VehicleprofileArmor> {
        return NSFetchRequest<VehicleprofileArmor>(entityName: "VehicleprofileArmor")
    }

    @NSManaged var front: NSDecimalNumber?
    @NSManaged var rear: NSDecimalNumber?
    @NSManaged var sides: NSDecimalNumber?
    @NSManaged var vehicleprofileArmorListHull: VehicleprofileArmorList?
    @NSManaged var vehicleprofileArmorListTurret: VehicleprofileArmorList?
}
